web/src/main/webapp/WEB-INF/classes/setup/sql/migrate/v2110/2-migrate-default.sql

```diff
@@ -51,7 +51,7 @@ UPDATE Settings SET name = 'system/shib/firstname', datatype = 0 WHERE id = 176;
 UPDATE Settings SET name = 'system/shib/profile', datatype = 0 WHERE id = 177;
 UPDATE Settings SET name = 'system/userSelfRegistration/enable', datatype = 2, internal = 'n' WHERE id = 191;
 UPDATE Settings SET name = 'system/clickablehyperlinks/enable', datatype = 2 WHERE id = 201;
-UPDATE Settings SET name = 'system/localrating/enable', datatype = 2 WHERE id = 211;
+UPDATE Settings SET name = 'system/localrating/enable', datatype = 0 WHERE id = 211;
```

Comment:
This change should not be in this file, but in the the last migration file, otherwise if a user had the database in 3.0.0 will not be upgraded properly.